menu/cakemenu.html

```diff
@@ -39,111 +35,68 @@
 										</header>
 										<p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue. Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit nullam.</p>
-										<ul class="actions">
-											<li><a href="../orders.html" class="button big">Order Now</a></li>
-										</ul>
 									</div>
 									<span class="image object">
-										<img src="../images/bio_pic.jpg" alt="Amanda next to two-tiered wedding cake" />
+										<img src="../images/pic01.jpg" alt="placeholder" />
 									</span>
 								</section>
 
-							<!-- Section -->
-								<!-- <section>
-									<header class="major">
-										<h2>Erat lacinia</h2>
-									</header>
-									<div class="features">
-										<article>
-											<span class="icon fa-gem"></span>
-											<div class="content">
-												<h3>Portitor ullamcorper</h3>
-												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
-											</div>
-										</article>
-										<article>
-											<span class="icon solid fa-paper-plane"></span>
-											<div class="content">
-												<h3>Sapien veroeros</h3>
-												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
-											</div>
-										</article>
-										<article>
-											<span class="icon solid fa-rocket"></span>
-											<div class="content">
-												<h3>Quam lorem ipsum</h3>
-												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
-											</div>
-										</article>
-										<article>
-											<span class="icon solid fa-signal"></span>
-											<div class="content">
-												<h3>Sed magna finibus</h3>
-												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
-											</div>
-										</article>
-									</div>
-								</section> -->
-
 							<!-- Section -->
 								<section>
 									<header class="major">
-										<h2>Cake Designs <span style="text-transform: uppercase; font-size: small;">(By Occasion)</span></h2>
+										<h2>Cakes</h2>
 									</header>
 									<div class="posts">
 										<article>
-											<a href="#" class="image"><img src="../images/cakes/cake_pic04.jpg" alt="kid's rodeo themed birthday cake" /></a>
-											<h3>Birthdays (Kids)</h3>
+											<a href="#" class="image"><img src="../images/cakes/cake_pic03.jpg" alt="zoo animal themed birthday cake" /></a>
```

Comment:
maybe you meant to save this TODO for later, but don't forget to change the `href` for the image to redirect to the same page as the `Order This` button

Reply:
good call, thanks for the reminder. I fixed the `href` for the images to redirect to the respective product pages in the following commit [6a7009e](https://github.com/Yu-HuanWu/mandybbakes/pull/22/commits/6a7009e6aef24bd156b1541cf275364f1669ac82) part of this PR.